src/scss/components/_upload-avatar-modal.scss

```diff
@@ -0,0 +1,29 @@
+.zoomandcrop {
```

Comment:
CSS classes should be camelCase, and new components should use CSS Modules instead of being added to the global stylesheet.

This ones on me for not warning about this. 

If you're new to the concept, A good example of using css modules would be something like the password fieldset under `src/components/Fieldsets/PasswordFieldset`